doc/calculator/Cargo.toml

```
@@ -6,14 +6,12 @@ build = "build.rs" # <-- We added this and everything after!
 workspace = "../.."
 edition = "2021"
 
-[build-dependencies.lalrpop]
-version = "0.19.10"
-path = "../../lalrpop"
-features = ["lexer"]
-
 [dependencies]
-regex = "1"
+lalrpop-util = { version = "0.19.10", path = "../../lalrpop-util" }
 
-[dependencies.lalrpop-util]
-version = "0.19.10"
-path = "../../lalrpop-util"
+[build-dependencies]
+lalrpop = { version = "0.19.10", path = "../../lalrpop", features = ["lexer"] }
+regex-syntax = { version = "0.6", default-features = false, features = ["unicode-perl"] }
+
+[dev-dependencies]
+regex = { version = "1.8", default-features = false, features = ["unicode-perl"] }
```

```suggestion
regex = "1.8"
```
Rather than making tutorial so heavy, how about just like this? The default feature is `unicode`, so it will be ok.

---

Since not having `unicode` enabled as a blanket feature (let alone default features), wouldn't that be a good opportunity to document it?